case-lib/lib.sh

```diff
@@ -85,3 +91,29 @@ func_lib_restore_pulseaudio()
     unset PULSECMD_LST
     declare -ag PULSECMD_LST
 }
+
+func_lib_trigger_stress()
+{
+    export LOG_ROOT=$LOG_ROOT
+    STRESS_OUTPUT_LOG="$LOG_ROOT/stress.txt"
```

Comment:
Can we have some iteration index here in the log files? Or this will contain full round stress test logs?

Reply:
because it have different stress type
So here I write for the reboot case, it will trigger this function after each time system reboot
But from the S3, I think this type case just to trigger this function once
So maybe add status control function in it?